Apps/phvmray/vmray.json

```diff
@@ -1,37 +1,41 @@
 {
     "appid": "cfac072d-79fa-406b-a2e1-25a2fd5785c4",
     "name": "VMRay",
-    "description": "Connector for VMRay Analyzer",
-    "publisher": " VMRay ",
+    "description": "This app enables you to detonate files and URLs, and perform investigative actions, using the VMRay Platform, thereby giving you automated analysis and advanced threat detection through an agentless hypervisor-based sandbox",
+    "publisher": " VMRay",
+    "contributors": [
```

Comment:
Please update the syntax of contributors key as mentioned below:
`"contributors": [
        {
            "name": "Juliya Smith"
        }
    ]`